<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateObjectsDialog</class>
 <widget class="QDialog" name="GenerateObjectsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate Objects</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="tabelLabel">
     <property name="text">
      <string>Select Table:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="tableComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="generateLabel">
     <property name="text">
      <string>Enter Number to Generate:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="generateEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GenerateObjectsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GenerateObjectsDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
